modules/ROOT/pages/installing-multiple-instances.adoc

```diff
@@ -0,0 +1,228 @@
+= Installing Multiple Instances of Runtime Fabric on a Single Cluster
+
+Installing multiple instances of Anypoint Runtime Fabric enables you to share the same cluster among multiple Runtime Fabrics, which helps you to use resources efficiently.
+
+[NOTE]
+To use this feature, you must upgrade to the minimum version of the agent that supports multiple instances.
+
+There will be 1:1 mapping for the agent namespace to the app namespace. Sharing app namespaces is not supported among multiple agent installations.
+
+== Install Multiple Instances of Runtime Fabric
+
+To install multiple instances of Anypoint Runtime Fabric on a single BYOK (Bring Your Own Kubernetes) cluster, follow these steps:
+
+. Create a Runtime Fabric instance using Runtime Manager.
+. Create a custom namespace to install the Runtime Fabric instance:
++ 
+----
+kubectl create ns <rtf_namespace>
```

Comment:
inconsistent again, we have different place holders in different places of docs

`<rtf-namepace>`
`<rtfNamespace>`
`<rtf_namespace>`

Let's pick one and stay consistent. My vote is for `<rtf-namespace>`. Same with app namespace, let's keep it consistent `<app-namespace>`